### DISCUSSION TITLE
월드 출시 이후 유저수 확인이 되지 않아요

### DISCUSSION THREAD

**Discussion by sunysiny** [naverz/zepeto-studio-kor#830] [5. Q&A]
안녕하세요
Jun 02, 2022 02:11 PM에 출시 완료된 월드맵에 유저들과 플레이를 진행하는데도
통계자료에 유저 수가 계속 0으로만 나오네요ㅠㅠ

확인 부탁드립니다
https://world.zepeto.me/ko/detail/AWan23wK87qt7ot1jWC9s5C?referrer=copylink_share

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요~ 월드 출시 축하드립니다!
방문자수는 실시간으로 올라가지 않습니다. 조금만 기다려 주세요~ 😄

**Reply by sunysiny** [reply]
앗 감사합니다:)
첫번째 월드는 주말 출시 후 하루 뒤쯤 통계 지표를 확인했어서...
실시간 업데이트 되는게 아닌줄 몰랐네용^^;;(민망민망)

모든 크리에이터 분들 화이팅입니다🥰

**Reply by sunysiny** [reply]
@Zepeto-TechEvangelist 
> 안녕하세요~ 월드 출시 축하드립니다! 방문자수는 실시간으로 올라가지 않습니다. 조금만 기다려 주세요~ 😄

안녕하세요~ 
스튜디오 내 콘텐츠 -> 월드 리스트에서 "유저" 수 카운팅이 어느 시점부터 업데이트 되지 않아요 ㅠㅠ
월드 2개는 여전히 0으로 되어 있고, 월드 1개는 업데이트가 안되는것 같습니다.

유저 방문 수 합계를 확인하려면 어떻게 해야 할까요?

**Reply by Zepeto-TechEvangelist** [reply]
안녕하세요, 제작하신 월드가 싱글 플레이 월드였다면 얼마전에 관련 문제가 있어서 수정 작업을 진행 하였습니다.
제페토 앱 3.14.0 이후 버전에서는 카운트가 정상 동작할 것으로 기대합니다.
감사합니다.

**Reply by sunysiny** [reply]
@Zepeto-TechEvangelist 
![image](https://user-images.githubusercontent.com/104808306/183577559-39f82cfb-f912-4675-8383-8e23237931bf.png)

멀티플레이 월드이고, 제페토 앱 내에서는 유저 수가 확인이 되는데, 웹 시스템에서는 조회가 되지 않습니다!ㅠ
첫번째 월드도 어느 시점부터 카운트가 제대로 되고 있지 않네요

**Reply by Zepeto-TechEvangelist** [reply]
안녕하세요, 해당 이슈 확인해 보겠습니다.

**Reply by Zepeto-TechEvangelist** [reply]
안녕하세요, 문의 주신 내용 확인해서 스튜디오 사이트에서 유저수가 보이지 않는 버그 수정하였습니다.
지금은 정상적으로 노출됨을 확인하였습니다. 
제보 감사드립니다.
